README.md

```diff
@@ -7,18 +7,30 @@
 
 _(Disclaimer: This project is currently under active development, and as such, all source code may not be included in any release. This means that the code is subject to change without notice, and that any information contained within the code should be considered as work in progress.)_
 
-Cloud-native technologies can be applied to Artificial Intelligence (AI) for scalable
-application in dynamic environments such as public, private and hybrid cloud. But
-it requires cloud native design to decompose monolithic inference pipeline into several
-microservices.
+## 1. Overview
+
+This project provides multiple-streams real-time inference pipeline based on cloud native design pattern as following architecture
+diagram:
 
 ![](docs/cnap_arch.png)
 
-The major microservices includes:
+Cloud-native technologies can be applied to Artificial Intelligence (AI) for scalable application in dynamic environments
+such as public, private and hybrid cloud. But it requires cloud native design to decompose monolithic inference pipeline
```

Comment:
change to: But it requires a cloud native design

Reply:
nice comment, fixed!